scripts/deploy/README.md

```diff
@@ -152,10 +152,13 @@ chmod +x ./deploy-webapi.sh
 
 # Deploy Hosted Plugins
 
-> **_NOTE:_** This step can be skipped if the previous Azure Resources creation step succeeded without errors. The `deployWebSearcherPackage = true` setting in main.bicep ensures that the WebSearcher is deployed.
+> **_NOTE:_** This step can be skipped if the required resources for the web searcher plugin are not deployed. The required resources include a Bing resource that may require an elevated permission, and an Azure Function. The required resources are NOT deployed by default. To deploy the required resources, use the `-DeployWebSearcherPlugin` or `--deploy-web-searcher-plugin` flag when running the **deploy-azure.ps1/deploy-azure.sh** script.
```

Comment:
```suggestion
> **_NOTE:_** This step can be skipped if the required resources for the web search plugin are not deployed. The required resources include a Bing Search resource and an Azure Function. The required resources are NOT deployed by default. To deploy the required resources, use the `-DeployWebSearcherPlugin` or `--deploy-web-searcher-plugin` flag when running the **deploy-azure.ps1/deploy-azure.sh** script.
```

Don't mention elevated permissions.  That's just something specific to how our subscription is set up. Using another subscription doesn't have this problem.